README.md

```diff
@@ -48,8 +48,9 @@ Design resources for Rspack, Rsbuild, Rspress and Rsdoctor.
 ### Rsbuild Starter
 
 - [rsbuild_vue3_h5_template](https://github.com/DMaiGit/rsbuild_vue3_h5_template): A project template for Vue 3. It includes popular libraries such as Axios, Pinia, Vant, and Vue Router.
+- [shark](https://github.com/h7ml/shark): A React project with Ant Design, Ant Design Pro Components, AntV G2Plot, ahooks, react-use, Axios, i18next, localforage, Mock.js, NProgress, and more.
```

Comment:
I found that shark is using `@rspack/cli` instead of Rsbuild, maybe it should be added to the `Rspack Starter` part.